go.mod

```diff
@@ -23,14 +23,15 @@ require (
 	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
 	github.com/xeipuuv/gojsonschema v1.1.0 // indirect
 	gopkg.in/gorp.v1 v1.7.2 // indirect
-	helm.sh/helm v3.0.0-alpha.2+incompatible
+	helm.sh/helm v3.0.0-beta.1+incompatible
```

Comment:
change to helm.sh/helm v3.0.0-beta.2+incompatible

Reply:
@slenky Please update to beta.2.